.github/workflows/chromatic.yml

```
@@ -1,30 +1,32 @@
+        run: npm ci  # Use 'yarn install --frozen-lockfile' if you use Yarn
 
       - name: Publish to Chromatic
-        uses: chromaui/action@latest
+        uses: chromaui/action@v1
         with:
-          # ⚠️ Make sure to configure a `CHROMATIC_PROJECT_TOKEN` repository secret
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
\ No newline at end of file
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          # Optional: Add any additional Chromatic CLI flags here
+          # For example: --exit-zero-on-changes
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)